src/error.rs

```diff
@@ -278,8 +278,19 @@ impl ErrorMessage {
             None => (Span::call_site(), Span::call_site()),
         };
 
-        // compile_error!($message)
+        // std::compile_error!($message)
         TokenStream::from_iter(vec![
+            TokenTree::Ident(Ident::new("std", start)),
```

Comment:
Please use `core` instead of `std`. Otherwise, it will generate an ugly error message in `#![no_std]` crate.